app.rb

```diff
@@ -0,0 +1,24 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+
+  configure :development do
+    register Sinatra::Reloader 
+  end
+
+  get '/' do
+    @peep = Peep.all
+    erb :homepage
+  end
+
+  post '/' do
+    @newpeep= Peep.create(peep: params[:peep])
```

Comment:
Do you get a successful result from this method call, to create a new peep? 

If not, if you were to add a line above this to print params (**p params**), do you see a peep message stored in params[:peep]?